swagger/additionalProperties.json

```diff
@@ -0,0 +1,294 @@
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PetAPInPropertiesWithAPString"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A Pet which contains more properties than what is defined.",
+            "schema": {
+              "$ref": "#/definitions/PetAPInPropertiesWithAPString"
+            }
+          },
+          "default": {
+            "description": "Unexpected error",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Error": {
+      "properties": {
+        "status": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "message": {
+          "type": "string"
+        }
+      }
+    },
+    "PetAPTrue": {
+      "required": [
+        "id"
+      ],
+      "properties": {
+        "id": {
+          "type": "integer"
+        },
+        "name": {
+          "type": "string"
+        },
+        "status": {
+          "type": "boolean",
+          "readOnly": true
+        }
+      },
+      "additionalProperties": true
+    },
+    "PetAPObject": {
+      "type": "object",
+      "required": [
+        "id"
+      ],
+      "properties": {
+        "id": {
+          "type": "integer"
+        },
+        "name": {
+          "type": "string"
+        },
+        "status": {
+          "type": "boolean",
+          "readOnly": true
+        }
+      },
+      "additionalProperties": {
+        "type": "object"
+      }
+    },
+    "PetAPString": {
+      "type": "object",
+      "required": [
+        "id"
+      ],
+      "properties": {
+        "id": {
+          "type": "integer"
+        },
+        "name": {
+          "type": "string"
+        },
+        "status": {
+          "type": "boolean",
+          "readOnly": true
+        }
+      },
+      "additionalProperties": {
+        "type": "string"
+      }
+    },
+    "PetAPInProperties": {
+      "type": "object",
+      "required": [
+        "id"
+      ],
+      "properties": {
+        "id": {
+          "type": "integer"
+        },
+        "name": {
+          "type": "string"
+        },
+        "status": {
+          "type": "boolean",
+          "readOnly": true
+        },
+        "additionalProperties": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "number"
+          }
+        }
+      }
+    },
+    "PetAPInPropertiesWithAPString": {
+      "type": "object",
+      "required": [
+        "id"
+      ],
+      "additionalProperties": {
+        "type": "string"
+      },
+      "properties": {
+        "id": {
+          "type": "integer"
+        },
+        "name": {
+          "type": "string"
+        },
+        "status": {
+          "type": "boolean",
+          "readOnly": true
+        },
+        "additionalProperties": {
```

Comment:
@olydis - Fun fact, modeler renames this to additionalProperties1 as this collides with outer additionalProperties, which is the right thing to do.